third_party/istio-1.3.6/download-istio.sh

```diff
@@ -15,34 +15,34 @@ tar xzf istio-${ISTIO_VERSION}-linux.tar.gz
 cd istio-${ISTIO_VERSION} || exit
 
 # Create CRDs template
-helm template --namespace=istio-system \
+helm template --name-template=${ISTIO_VERSION} --name-template=${ISTIO_VERSION} --namespace=istio-system \
```

Comment:
Is the `--name-template` argument duplicated by design?

Reply:
Thanks for catching. Nope this is a mistake. Fixed.